airbyte-integrations/connectors/source-planhat/source_planhat/run.py

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
```

Comment:
question: Does Airbyte hold the copyright? Or do we? Is this supposed to be 2024?

Probably not that important but asking to at least have a discussion about it.

Reply:
It's MIT anyway, and if you want to keep this in the fork, you can use yours. But, I want to merge that source into Airbyte today anyway — perhaps you can use the one from upstream instead of the whole PR?